src/OrchardCore/OrchardCore.DisplayManagement.Abstractions/Extensions/ResultExecutingContextExtensions.cs

```diff
@@ -0,0 +1,9 @@
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace Microsoft.AspNetCore.Mvc.Filters;
+
+public static class ResultExecutingContextExtensions
+{
+    public static bool IsViewOrPageResult(this ResultExecutingContext context)
```

Comment:
I might prefer to add two extensions `IsViewResult()` & `IsPageResult()` then you can apply "or" in the condition when it's needed

Reply:
@hishamco anyway, I added them.

Reply:
So no need for ` IsViewOrPageResult()` anymore :)

Reply:
`IsViewOrPageResult()` simply the code in all the filters.

Reply:
sorry @hishamco I removed what I consider useless `IsViewResult` and `IsPageResult` since it encouraged you to use them over the simpler `IsViewOrPageResult`